packages/cli/src/generated/synthql/db.ts

```diff
@@ -0,0 +1,60 @@
+// This file was generated by @synthql/backend
+import { query } from '@synthql/queries';
```

Comment:
@jimezesinachi this shouldn't have been commited

Reply:
Pay more attention to your diffs. This is a sign that you're not reviewing your own code.

Reply:
I was thinking the same, but I did when I noticed the generated files had been committed in the backend package.

Reply:
gotcha. In the case of other files we do rely on them for testing, but not here.

Reply:
Deleted!